disnake/interactions/base.py

```diff
@@ -172,6 +175,7 @@ class Interaction:
     )
 
     def __init__(self, *, data: InteractionPayload, state: ConnectionState):
+        self.data: Mapping[str, Any] = data["data"]
```

Comment:
Sorry about the previous comment, this *can* be optional.
```suggestion
        self.data: Mapping[str, Any] = data.get("data") or {}
```